python/pyspark/streaming/flume.py

```diff
@@ -54,8 +54,13 @@ def createStream(ssc, hostname, port,
         :param bodyDecoder:  A function used to decode body (default is utf8_decoder)
         :return: A DStream object
 
-        .. note:: Deprecated in 2.3.0
+        .. note:: Deprecated in 2.3.0. Flume support is deprecated as of Spark 2.3.0.
+            See SPARK-22142.
         """
+        warnings.warn(
```

Comment:
Seems `warnings` is not imported in this file?

Reply:
Yes, it is not. Will make a followup after double checking other files too. Thank you.

Reply:
thank you :) It will be good to also check why master build does not fail since python should complain about it.

Reply:
Sure,  I took a quick look and I think this one is actually not being tested and seems that's why .. will double check and take a closer look tonight (KST).

I have seen few mistakes about this so far and .. I am working on Python coverage BTW - https://issues.apache.org/jira/browse/SPARK-7721

Anyway, it was my stupid mistake. Thanks ..